### PR TITLE
Fix/fix tests docker

### PR DESCRIPTION
**Pull request by rmoch**

### CODE REVIEW

File: fun/envs/lms/unittests.py [hunk 1]
```diff
@@ -7,6 +7,12 @@
 import logging
 logging.getLogger("backoffice.views").setLevel(logging.ERROR)
 logging.getLogger("edxmako.shortcuts").setLevel(logging.ERROR)
+logging.getLogger("openedx.core.djangoapps.content.course_overviews.models").setLevel(logging.ERROR)
+logging.getLogger("py.warnings").setLevel(logging.ERROR)
+logging.getLogger("backoffice.utils_proctorU_api").setLevel(logging.ERROR)
+logging.getLogger("elasticsearch").setLevel(logging.ERROR)
+logging.getLogger("urllib3.connectionpool").setLevel(logging.ERROR)
+logging.getLogger("payment_api.views").setLevel(logging.ERROR)
```
Comment (jmaupetit) [inline fun/envs/lms/unittests.py]: This should be configured in Test environment settings.

File: setup.cfg [hunk 2]
```diff
@@ -33,6 +33,7 @@ install_requires =
     easy-thumbnails==2.2
     edx-gea==0.2.0
     glowbl-xblock==0.2.0
+    oauth2client==3.0.0
```
Comment (jmaupetit) [inline setup.cfg]: Also add a comment here to explain why it must be installed before `google-api-python-client` (and thus why you are not respecting alphabetical order :stuck_out_tongue:).

File: fun/envs/lms/unittests.py (hunk 1 above)
Comment (rmoch) [reply fun/envs/lms/unittests.py]: What do you mean ? This is the Django settings file to use to run unit tests

Comment (jmaupetit) [reply fun/envs/lms/unittests.py]: You logs configuration should stand in the settings used with the test runner: https://docs.djangoproject.com/en/1.11/topics/logging/#configuring-logging

**Pull request merged**